.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: fhir2dataset
+
+on: [push]
+
+jobs:
+  build:
```

Comment:
don't you want to add a job to publish on pypi ? there is an example on pyfhirstore

Reply:
Done ! :smile: